src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMappingTest.java

```diff
@@ -399,8 +381,115 @@ void getHearingLocations() {
     @DisplayName("When override Hearing Venue Epims Ids is not empty getHearingLocations returns the override values")
     @Test
     void getHearingLocationsOverride() {
+        SscsCaseData caseData = buildOverrideHearingLocations();
 
-        SscsCaseData caseData = SscsCaseData.builder()
+        List<HearingLocation> result = HearingsDetailsMapping.getHearingLocations(caseData, referenceDataServiceHolder);
+
+        assertThat(result)
+            .hasSize(2)
+            .extracting("locationId","locationType")
+            .containsExactlyInAnyOrder(
+                tuple("219164", COURT),
+                tuple("436578", COURT));
+    }
+
+    @DisplayName("When a case has been adjourned and a different venue has been selected, return the new venue")
+    @Test
+    void getHearingLocationsAdjournmentNewVenue() {
+        SscsCaseData caseData = buildOverrideHearingLocations();
+
+        caseData.getAppeal().setHearingSubtype(HearingSubtype.builder()
+                                                   .wantsHearingTypeTelephone("yes")
+                                                   .hearingTelephoneNumber("07483871426").build());
+        caseData.getAppeal().setHearingOptions(HearingOptions.builder()
+                                                   .wantsToAttend("yes").build());
```

Comment:
Please remove all excessive indents in this file. I think this will have to be done manually as the automatic formatting in intellij leads to this layout which is hard to read.

Reply:
Done